src/problems/sudoku.clj

```diff
@@ -0,0 +1,124 @@
+(ns problems.sudoku)
+;;OH BOY KIDS HERE WE GO
```

Comment:
There's a weird inconsistency in the tone in your code. Most of it looks very professional, with nice things like doc strings, but then you have lines like this that really don't fit the tone you're trying to set.